js/ui/switch.js

```diff
@@ -230,10 +233,14 @@ var Switch = Editor.inherit({
             onStart: this._swipeStartHandler.bind(this),
             onUpdated: this._swipeUpdateHandler.bind(this),
             onEnd: this._swipeEndHandler.bind(this),
-            itemSizeFunc: this._getMarginBound.bind(this)
+            itemSizeFunc: this._getItemSizeFunc.bind(this)
         });
     },
 
+    _getItemSizeFunc: function() {
+        return this._$switchContainer.outerWidth(true) - parseFloat(window.getComputedStyle(this._$handle.get(0)).width);
```

Comment:
Are any disadvantages to using `element.getBoundingClientRect().width` instead of `parseFloat(window.getComputedStyle(element).width)`